src/component/FilterSection.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { useProduct } from "../context/product-context";
+import "../App.css";
+function FilterSection() {
+  const { state, dispatch } = useProduct();
```

Comment:
give descriptive name to the state and dispatch, like `productsState, productsDispatch` since there will be multiple such states and dispatches in a single file

Reply:
Done, Now given a specific name.